code/game/machinery/doors/multi_tile.dm

```diff
@@ -317,6 +321,7 @@
 	opacity = FALSE
 	glass = TRUE
 	var/throw_dir = EAST
+	queen_pryable = FALSE
```

Comment:
```suggestion
	queen_pryable = FALSE
	var/throw_dir = EAST
```
im gonna seem like the ultimate pedant but var overrides should come before new var declarations

Reply:
Ah, this is also one of my pet peeves I'm amazed I missed it